lib/departure/migration.rb

```diff
@@ -0,0 +1,96 @@
+module Departure
+  # Hooks Percona Migrator into Rails migrations by replacing the configured
```

Comment:
this is a good chance to rename `Percona Migrator` to `Departure`. This naming is pretty old already.